src/ansys/fluent/core/fluent_connection.py

```diff
@@ -324,9 +489,57 @@ def get_fluent_version(self):
         """Gets and returns the fluent version."""
         return self.scheme_eval.version
 
-    def exit(self) -> None:
-        """Close the Fluent connection and exit Fluent."""
-        self._finalizer()
+    def exit(self, force: bool = None) -> None:
```

Comment:
If we have the definition as `exit(self, timeout=None)`, `PYFLUENT_FORCE_EXIT` and `TIMEOUT_TO_FLUENT_KILL` can be combined to single env. `PYFLUENT_TIMEOUT_TO_FLUENT_KILL`.

Reply:
I think that is a great idea, thanks Mainak